resources/titles.xml

```diff
@@ -6429,6 +6437,24 @@
       <pubPlace>Bonn</pubPlace>
       <date when="2000">2000</date>
    </bibl>
+   
+   <bibl type="m" xml:id="ObjectBiographies">
```

Comment:
As a rule, we now make xml:ids for bibliography entries on the form "author-surname_year"; since you've already used this, leave it for now, but for the future try to follow that structure.